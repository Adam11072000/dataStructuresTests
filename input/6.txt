init 48
addImage 121
mergeSuperPixels 96 12 49
getHighestScoredLabel 121 44
deleteImage 121
addImage 197
addImage 10
getHighestScoredLabel 197 37
mergeSuperPixels 10 19 0
addImage 10
addImage -5
deleteImage 61
resetLabelScore 197 11 0
quit