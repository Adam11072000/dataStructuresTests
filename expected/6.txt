init done.
addImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
getHighestScoredLabel: FAILURE
deleteImage: SUCCESS
addImage: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
addImage: FAILURE
addImage: INVALID_INPUT
deleteImage: FAILURE
resetLabelScore: INVALID_INPUT
quit done.
